storage/object/how-to/get-started-edge-services.mdx

```diff
@@ -0,0 +1,293 @@
+
+3. Click the **Edge Services** tab.
+
+    TODO SCREENSHOT
+
+4. In the **Endpoint** panel, click **Edit**. The following screen displays:
+
+    TODO SCREENSHOT
+
+5. Edit the subdomain as desired - do not forget to also set up a new CNAME record if necessary. 
+
+6. Select a different certificate from Secret Manager, or manually import a new certificate for your custom domain.
+
+7. Click **Edit domain** to finish.
+
+### How to reset your customized domain to the original Edge Services endpoint
+
+Even though the original Edge Services endpoint (e.g. `https://bucket-name.edge.scw.cloud`) will continue to work after you add a customized domain, you can choose to remove your customized domain completely and go back to the original Edge Services endpoint only. This is done via the reset function:
+
+1. Click **Object Storage** in the **Storage** section of the Scaleway console side menu. The list of your buckets displays.
+
+2. Click the name of the bucket you want to configure a custom domain for.
+
+3. Click the **Edge Services** tab.
+
+    TODO SCREENSHOT
+
+4. In the **Endpoint** panel, click **Reset**. 
+
+    A screen displays warning you that this will reset the bucket's domain back to the default Edge Services endpoint. Edge Services will consider your customized subdomain as unknown. You should also remember to delete your CNAME record from your domain provider, and your SSL/TLS certificate.
+
+5. Click **Reset domain** to finish.
+
+## How to configure your cache
+
+The cache feature allows you to cache your bucket's content with Edge Services. This means that content can be served directly to users from Edge Services' servers, enhancing performance. 
+
+You can disable and enable caching at will, as well as control the lifetime of an object in the cache. You can also purge your entire cache, or specific objects within it. A log is displayed to help you track your purge events.
+
+### How to enable your cache
+
+1. Click **Object Storage** in the **Storage** section of the Scaleway console side menu. The list of your buckets displays.
+
+2. Click the name of the bucket you want to configure a cache for.
+
+3. Click the **Edge Services** tab.
+
+    TODO SCREENSHOT
+
+4. In the **Cache** panel, use the <Icon name="toggle" /> icon to enable the cache.
+
+    The **Life time** configuration box displays. This enables you to define, in seconds, how long an object can be stored in the cache before it must be retrieved freshly from the bucket.
+
+    <Message type="tip">
+    As an example, a value of 0 means that objects will not be cached, unless they have a separately-defined caching directive.
+    </Message>
+
+5. Leave the default value of 3600s (1 hour) in place, or enter another value.
+
+The cache is now enabled.
+
+### How to purge all objects from your cache
+
+This clears all objects from your cache. Afterwards, fresh copies must be retrieved from the bucket before Edge Services can store them again in the cache. 
```

Comment:
@RoRoJ 

>  fresh copies must be retrieved from the bucket

I fear that saying "must" implies that the user has to do something. This is a completely automated process carried by the edge services node whenever a request for an object that's not in the cache comes in.